plugins/server/ai.koog/koog/versions.ktor.yaml

```
@@ -0,0 +1 @@
+"[2.0,)": ai.koog:koog-ktor:0.4.+
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)